app/tauri/src/main.rs

```diff
@@ -49,6 +54,9 @@ fn main() {
 
             println!("Pomatez is ready");
         }
+        RunEvent::ExitRequested { api, .. } => {
+            api.prevent_exit();
```

Comment:
I thought I'd already implemented the closing to tray in the settings.

Or are you talking about the Mac's close to tray? If so, you'll need to add a check or a compiler flag to make sure it doesn't do this on Windows as that is not the expected behavior.

Reply:
How is this implemented for these other platforms on tauri? I cannot find any code that handles this for any platform

Reply:
I don't believe they handle it themselves, I believe tauri just provides a platform to make apps. The same as electron doesn't implement this behavior either.